### DISCUSSION TITLE
How do I configure starship with fish shell?

### DISCUSSION THREAD

**Discussion by DirtFrog** [prasanthrangan/hyprdots#908] [Q&A] [answered]
Hi, I just installed hyprdots and my starship prompt is just the default one. I tried putting my starship.toml in .config and sourcing fish but it's still stuck at default.

Where should I be putting my starship.toml? Is the default starship prompt normal behavior for the install?

**Comment by kRHYME7** [comment]
Check this out : https://github.com/prasanthrangan/hyprdots/issues/566

So far, this is the config that is in the repo. the default starship is used. Maybe see starship repo on how to configure it.

**Reply by DirtFrog** [reply, marked as answer]
I was being dumb, misspelt starship when moving it back into .config

**Reply by kRHYME7** [reply]
Would you care to  share some starship configs? I am staring to use fish for terminal as it's convenient.

**Reply by DirtFrog** [reply]
Ya for sure. I found this template a while back and modified it a bit. Uploaded as a txt but should be .toml
[starship.txt](https://github.com/prasanthrangan/hyprdots/files/14334961/starship.txt)

It uses nerd fonts for some symbols
